src/werkzeug/serving.py

```diff
@@ -445,9 +447,26 @@ def get_header_items(self):
             # W3C RFC 2616 Section 4.2.
             items = []
             for header in self.headers.headers:
-                # Remove "\n\r" from the header and split on ":" to get
+                # Remove "\r\n" from the header and split on ":" to get
                 # the field name and value.
-                key, value = header[0:-2].split(":", 1)
+                try:
+                    key, value = header[0:-2].split(":", 1)
+                except ValueError as e:
+                    # If header could not be slit with : but starts with white
+                    # space and it follows an existing header, it's a folded
+                    # header.
+                    if header[0] in ("\t", " ") and len(items) > 0:
```

Comment:
`... and items:`

Reply:
`header[0] in "\t "` would work as well (but maybe the tuple is slightly more readable)

Reply:
Agreed, I tried it both ways and using the tuple seemed more explicit that it needed to be one of the values.

Reply:
Updated to the more Python `... and items`: